### PR TITLE
Generate tests with the real names

### PR DESCRIPTION
**Pull request by ybbbby**
Replace uuid-based LoadTest names with the real names

### CODE REVIEW

File: tools/run_tests/performance/loadtest_config.py
Comment (paulosjca) [inline tools/run_tests/performance/loadtest_config.py:68]: I would keep the test, replacing the value 63 with 253 (in case someone specifies a really long prefix).

See https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names.

**Review by paulosjca — COMMENTED** [review]
Please waiit until grpc/test-infra release 0.6.0 is complete before merging this.

File: tools/run_tests/performance/loadtest_config.py
Comment (ybbbby) [reply tools/run_tests/performance/loadtest_config.py:68]: Make sense. Done.

**Review by jtattermusch — APPROVED** [review]
LGTM once you have a detailed review from someone.

**Review by paulosjca — APPROVED** [review]

**Comment by paulosjca** [comment]
Release 0.6.1 of the benchmarking infrastructure is deployed, so merging this PR.

**Pull request merged**
